python/ct/crypto/verify_test.py

```diff
@@ -20,33 +20,15 @@ def read_testdata_file(test_file):
     with open(os.path.join(FLAGS.testdata_dir, test_file), 'rb') as f:
         return f.read()
 
-class LogVerifierTest(unittest.TestCase):
-    default_sth = client_pb2.SthResponse()
-    default_sth.tree_size = 42
-    default_sth.timestamp = 1348589667204
-    default_sth.sha256_root_hash = (
-        "18041bd4665083001fba8c5411d2d748e8abbfdcdfd9218cb02b68a78e7d4c23"
-        ).decode("hex")
-
-    default_sth.tree_head_signature = (
-        "040300483046022100befd8060563763a5e49ba53e6443c13f7624fd6403178113736e"
-        "16012aca983e022100f572568dbfe9a86490eb915c4ee16ad5ecd708fed35ed4e5cd1b"
-        "2c3f087b4130").decode("hex")
-
-    default_key_info = client_pb2.KeyInfo()
-    default_key_info.type = client_pb2.KeyInfo.ECDSA
-    default_key_info.pem_key = (
-        "-----BEGIN PUBLIC KEY-----\nMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAES0AfBk"
-        "jr7b8b19p5Gk8plSAN16wW\nXZyhYsH6FMCEUK60t7pem/ckoPX8hupuaiJzJS0ZQ0SEoJ"
-        "GlFxkUFwft5g==\n-----END PUBLIC KEY-----\n")
 
+class LogVerifierTest(object):
     def test_verify_sth(self):
-        verifier = verify.LogVerifier(LogVerifierTest.default_key_info)
-        self.assertTrue(verifier.verify_sth(LogVerifierTest.default_sth))
+        verifier = verify.LogVerifier(self.default_key_info)
```

Comment:
That's inherited from the original @ekasper code, but I'll see what better name I can come up with. Perhaps "{sth,key_info}_fixture"?